README.md

```diff
@@ -36,3 +36,45 @@ npm run build
 You can preview the production build with `npm run preview`.
 
 > To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
+
+---
+
+# Actual README Content
```

Comment:
I wouldn't mind wiping out the default svelte readme text.

Reply:
done :)